src/styles/themes/_default-theme.scss

```diff
@@ -313,6 +313,10 @@
   --warning-hover-color: #ce811a;
   --warning-hover-background-color: #ffe3b0;
   --warning-hover-border-color: #c97e19;
+  --warning-background1-color: var(--orange-background1-color); // Surface 1
+  --warning-background2-color: var(--orange-background2-color); // Surface 2
+  --warning-background3-color: var(--orange-background3-color); // Surface 3
```

Comment:
Will need UX input on this.

Reply:
Sounds good, I will discuss with David T.

Reply:
I discussed and showed this to David T. and he approves of these colors as this correlates directly with how disruptive colors use the red color theming:

`--disruptive-backgroundN-color` => `--red-backgroundN-color`
`--warning-backgroundN-color` => `--orange-backgroundN-color`

![image](https://github.com/EightfoldAI/octuple/assets/102243974/fcfc27d6-d2f3-4fad-8efb-65fef30d8655)
 
I am mimicking this related for warning:
![image](https://github.com/EightfoldAI/octuple/assets/102243974/239ec793-acdf-4175-a806-46010dcd38bc)
![image](https://github.com/EightfoldAI/octuple/assets/102243974/b6277421-0bf3-4ae9-8961-b81f6f963f5e)